### ISSUE TITLE
Maternal antibodies to platelet glycoproteins

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11297]
**Preferred term label:**
Maternal antibodies to platelet glycoprotein Ia
Maternal antibodies to platelet glycoprotein IIb
Maternal antibodies to platelet glycoprotein IIIa

**Synonyms**
Genes in order with OMIM:
ITGA2    https://omim.org/entry/621267
ITGA2B  https://omim.org/entry/621266
ITGB3    https://omim.org/entry/621264
**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
See 3 OMIM entries above

**Your nano-attribution (ORCID)**
